concepts/evaluation/queue.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Annotation Queues"
+---
+
+The Literal AI platform helps you organise and smoothen the human evaluation process by easily setting-up annotation queues.
+Each queue let you add data for review and provides an accessible and efficient ui for domain experts to score, tag and add to datasets crucial insights.
+
+## Creating an annotation queue
+
+You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" cta in the top right corner of the table.
+
+<img src="/images/create-annotation-queue.png" alt="Create annotation queue image" />
+
+This will open a modal form, that you can fill with a **name** to identify the annotation queue and a **description** to convey the queue meaning.
+
+<img src="/images/create-annotation-queue-form.png" alt="Create annotation queue image form" />
+
+<Note>By default each project comes with a "Default" annotation queue.</Note>
+
+## Populating a queue with data
+
+With your queues created you can manually add items to review.
+Accessing your run/generation logs, select them and insert them into the queue.
+
+<img src="/images/add-generations-queue.png" alt="Add generations to the queue" />
+
+To more precisely pick the relevant data you are able to select specific steps.
+
+<img src="/images/add-step-queue.png" alt="Add steps to the queue" />
+
+## Review queues
+
+On the "Annotation Queues" page, you can open a specific queue, allowing users to start reviewing added items.
+
+A reviewer can score and tags providing you with human feedbacks.
+
+<img src="/images/annotation-item-view.png" alt="The annotation item page" />
+
+A reviewer can also add a specific step data to an existing dataset.
+
+Finally, once the work is done, the item can be marked as reviewed.
```

Comment:
In general, passive voice weakens the writing by omitting the subject. "X can do Y to Z" is considered stronger than "Z can be Yed".

```suggestion
A reviewer can also add a specific step to an existing dataset for further processing.

Finally, once the work is done, they can mark the item as Reviewed to remove it from the Annotation Queue.
```

Reply:
This will not remove from the queue, it will keep the item but marked it as reviewed

Reply:
my bad, updated the suggestion